core/src/main/java/tc/oc/pgm/goals/ShowOptions.java

```diff
@@ -0,0 +1,45 @@
+package tc.oc.pgm.goals;
+
+import java.util.HashSet;
+import java.util.Set;
+import org.jdom2.Element;
+import tc.oc.pgm.util.xml.InvalidXMLException;
+import tc.oc.pgm.util.xml.XMLUtils;
+
+public class ShowOptions {
+
+  public enum ShowFlag {
+    SHOW_MESSAGES("show-messages"),
+    SHOW_EFFECTS("show-effects"),
+    SHOW_INFO("show-info"),
+    SHOW_SIDEBAR("show-sidebar"),
+    STATS("stats");
+
+    private final String name;
+
+    ShowFlag(String name) {
+      this.name = name;
+    }
+  }
+
```

Comment:
Should probably be `EnumSet` instead of `Set` everywhere in this class

Reply:
Ops I clicked the wrong lines here, I blame mobile!!!

Reply:
I think using the Set interface is ok, any set will work so there's no need to restrict it

However, the parse implementation *should* be initializing an enum set instead of a `new HashSet<>()`
The constructor is private so the only way to create these is thru parse anyways, so that will accomplish the same goal

Reply:
True! As long as the initializer is changed it should work fine

Reply:
Excellent, will have the parse implementation initialize as an enum set 👍